scripts/ci/ci_run_airflow_testing.sh

```diff
@@ -79,36 +79,3 @@ done
 
 RUN_INTEGRATION_TESTS=${RUN_INTEGRATION_TESTS:=""}
 
-if [[ ${RUNTIME:=} == "kubernetes" ]]; then
-    export KUBERNETES_MODE=${KUBERNETES_MODE:="git_mode"}
-    export KUBERNETES_VERSION=${KUBERNETES_VERSION:="v1.15.3"}
-
-    set +u
-    # shellcheck disable=SC2016
-    docker-compose --log-level INFO \
-      -f "${MY_DIR}/docker-compose/base.yml" \
-      -f "${MY_DIR}/docker-compose/backend-${BACKEND}.yml" \
-      -f "${MY_DIR}/docker-compose/runtime-kubernetes.yml" \
-      "${INTEGRATIONS[@]}" \
-      "${DOCKER_COMPOSE_LOCAL[@]}" \
-         run airflow \
-           '/opt/airflow/scripts/ci/in_container/entrypoint_ci.sh "${@}"' \
-           /opt/airflow/scripts/ci/in_container/entrypoint_ci.sh "${@}"
-         # Note the command is there twice (!) because it is passed via bash -c
-         # and bash -c starts passing parameters from $0. TODO: fixme
-    set -u
-else
-    set +u
-    # shellcheck disable=SC2016
-    docker-compose --log-level INFO \
-      -f "${MY_DIR}/docker-compose/base.yml" \
-      -f "${MY_DIR}/docker-compose/backend-${BACKEND}.yml" \
-      "${INTEGRATIONS[@]}" \
-      "${DOCKER_COMPOSE_LOCAL[@]}" \
-         run airflow \
-           '/opt/airflow/scripts/ci/in_container/entrypoint_ci.sh "${@}"' \
-           /opt/airflow/scripts/ci/in_container/entrypoint_ci.sh "${@}"
-         # Note the command is there twice (!) because it is passed via bash -c
-         # and bash -c starts passing parameters from $0. TODO: fixme
-    set -u
```

Comment:
This (`else` part) should still remain I think

Reply:
How come? We're not running those travis tests anymore

Reply:
I thought only the "if" part ran Kubernetes tests 

cc @potiuk Can you verify if we need the else part or is it safe to remove, please :)

Reply:
We definitely need the else. all tests are using it but the kubernetes ones.